axelrod/data/ann_weights.csv

```diff
@@ -0,0 +1,5 @@
+# name, features, hidden_layer_size, weights...
```

Comment:
Fine to leave for later, you could have the max number of columns with headers and have NANs in the other ones? (Not suggesting that for now).